skxray/tests/test_core.py

```diff
@@ -538,6 +543,34 @@ def run_image_to_relative_xyi_repeatedly():
             print('{0} calls successful'.format(num_calls))
 
 
+def test_radial_integration():
```

Comment:
This test is mostly a smoke test.  Can you add a test where the expected value is not all ones?